articat/config.py

```diff
@@ -162,7 +162,8 @@ def fs_tmp_prefix(self) -> str:
             r = tempfile.mkdtemp(suffix="artifact_temp_dir_")
             self._set_option(self._config, "fs", "tmp_prefix", r)
             warnings.warn(
-                f"FSArtifact temp directory not configured, assuming local mode, using temp directory: {r}"
+                f"FSArtifact temp directory not configured, assuming local mode, using temp directory: {r}",
+                stacklevel=1,
```

Comment:
🟢 what do these `stacklevel` args do? Is there anything about this that we should keep in mind or bring with us to the facets repo?

Reply:
`stacklevel=1` is the default, so stuck with that. Needed to add to quiet [no-explicit-stacklevel (B028)](https://docs.astral.sh/ruff/rules/no-explicit-stacklevel/). I didn't think more verbose warnings with an additional level of stack trace were necessary here, but also did not really think about it.